ui/components/MyProfile/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { SearchIcon } from '@heroicons/react/outline'
+import NFTCard from '@components/Shared/NFTCard'
+import Jazzicon, { jsNumberForAddress } from 'react-jazzicon'
+
+import { mockExplore } from '@components/mockData.js'
+
+const Home = () => {
```

Comment:
we should change the name of component to be MyProfile. Same for other files where this occurs